_overviews/scala3-scaladoc/index.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: multipage-overview
+title: Scaladoc
+partof: scala3-scaladoc
+num: 1
+next-page: docstrings
+---
+
+![scaladoc logo]({{ site.baseurl }}/resources/images/scala3/scaladoc/logo.png)
```

Comment:
This seems to be the old logo. Here's the new one https://github.com/lampepfl/dotty/blob/master/scaladoc/resources/dotty_res/images/scaladoc_logo.svg

Reply:
Changed